spec/controllers/anonymous_feedback_controller_spec.rb

```diff
@@ -29,11 +29,11 @@
     context "on the first page" do
       it "should show no results" do
         stub_support_api_anonymous_feedback(
-          { path_prefix: "/a" },
+          { path_prefixes: ["/a"] },
           "results" => [], "pages" => 0, "current_page" => 1,
         )
 
-        get :index, params: { "path" => "/a", "format" => "json" }
+        get :index, params: { "paths" => ["/a"], "format" => "json" }
```

Comment:
I think throughout this test we should use `paths: "<comma separated string>"` instead of `paths: [<array of strings>]` because the UI will generate the former, not the latter.

Reply:
Changed